.ci_support/osx_.yaml

```diff
@@ -1,5 +1,5 @@
 MACOSX_DEPLOYMENT_TARGET:
-- '10.14'
+- '10.9'
```

Comment:
Interesting. Why was 10.14 used here?

Reply:
See the discussion in https://github.com/conda-forge/cmake-feedstock/pull/98

Reply:
Thanks for the context. Also nice work on finding this solution! 😄